mods/ra2/sequences/misc.yaml

```diff
@@ -56,19 +56,17 @@ beacon:
 		Length: 1
 		ZOffset: 2047
 
-crate-effects: #Temp Null Some SHPs cannot be read by openra at this time, expect more
```

Comment:
You deleted the whole line instead of just the comment.

Reply:
Fixed.